<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-10</y>
      <width>1291</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(24, 24, 24);</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>120</y>
       <width>401</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(50, 50, 50);
border-radius: 25;</string>
     </property>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>470</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(127, 127, 127);
border-radius: 25;</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>0</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	font: 500 12pt &quot;AutoText&quot;;
}
QPushButton:hover
{
	color: rgb(194, 194, 194);
}</string>
          </property>
          <property name="text">
           <string>Log in</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	font: 500 12pt &quot;AutoText&quot;;
}
QPushButton:hover
{
	color: rgb(194, 194, 194);
}</string>
          </property>
          <property name="text">
           <string>Sign in</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="widget_4" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>461</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>91</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 550 20pt &quot;AutoText&quot;;
color: rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>350</y>
           <width>341</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: rgb(255, 81, 0);
	color: rgb(255, 255, 255);
	font: 600 12pt &quot;AutoText&quot;;
	border-radius: 5;
}
QPushButton:hover
{
	background-color: rgb(255, 204, 120);
	color: rgb(153, 153, 153);
}
QPushButton:pressed
{
	background-color: rgb(255, 191, 0);
}</string>
         </property>
         <property name="text">
          <string>Log in</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 500  9pt &quot;AutoText&quot;;
color: rgb(204, 204, 204)</string>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>341</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Your own password</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 500  9pt &quot;AutoText&quot;;
color: rgb(204,204,204);</string>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>341</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>username@gmail.com</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>232</x>
           <y>295</y>
           <width>131</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font: 500  8pt &quot;AutoText&quot;;
	color: rgb(204,204,204);
	text-decoration: underline;
}
QPushButton:hover
{
	color: rgb(125, 125, 125);
}
QPushButton:pressed
{
	color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>Forgot password?</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="widget_5" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>461</height>
         </rect>
        </property>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>390</y>
           <width>341</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: rgb(255, 81, 0);
	color: rgb(255, 255, 255);
	font: 600 12pt &quot;AutoText&quot;;
	border-radius: 5;
}
QPushButton:hover
{
	background-color: rgb(255, 204, 120);
	color: rgb(153, 153, 153);
}
QPushButton:pressed
{
	background-color: rgb(255, 191, 0);
}</string>
         </property>
         <property name="text">
          <string>Sign in</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 500  9pt &quot;AutoText&quot;;
color: rgb(204, 204, 204)</string>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>225</y>
           <width>341</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>username@gmail.com</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>195</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 500  9pt &quot;AutoText&quot;;
color: rgb(204,204,204);</string>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>341</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Your name</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>280</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 500  9pt &quot;AutoText&quot;;
color: rgb(204,204,204);</string>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>310</y>
           <width>341</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Your own password</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>101</width>
           <height>78</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 550 20pt &quot;AutoText&quot;;
color: rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string>Signin</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-40</y>
       <width>261</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../../Downloads/food-ordering-app-logo-with-points-and-fork-shapes-in-the-center-free-vector-removebg-preview.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>471</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../diary app/fast-food-board-with-hamburger-french-fries-isolated-on-transparent-background-free-png.webp</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
